pkg/datastoresrp/api/v20231001preview/mongodatabase_conversion_test.go

```diff
@@ -296,6 +296,11 @@ func TestMongoDatabase_ConvertDataModelToVersioned(t *testing.T) {
 					Username:             to.Ptr(""),
 					Status: &ResourceStatus{
 						OutputResources: nil,
+						Recipe: &RecipeStatus{
+							TemplateKind:    to.Ptr("bicep"),
+							TemplatePath:    to.Ptr("br:sampleregistry.azureacr.io/radius/recipes/abc"),
+							TemplateVersion: nil,
+						},
```

Comment:
Should this use `MustPopulateResourceStatusWithRecipe`?

Reply:
In line 298, we want outputresources = nil. MustPopulateResourceStatusWithRecipe expects an output resource and so did not use it.